.github/workflows/nightly.yaml

```diff
@@ -75,22 +75,19 @@ jobs:
   # This job is used to update the list of validated versions
   linter_tests_release:
     name: Plugin Tests Release
-    # runs-on: [self-hosted, "${{ matrix.os }}"] TODO(Tyler): Set after Windows self-hosted are established.
     runs-on: ${{ matrix.os }}
     timeout-minutes: 150
     strategy:
       fail-fast: false
       matrix:
         linter-version: [Snapshots, Latest]
-        os: [ubuntu-latest, macOS, windows-latest]
+        os: [ubuntu-latest, macOS]
```

Comment:
This has been failing rather noisily. Until we can invest time to burn it down, I'm disabling it